data-prepper-plugins/drop-processor/src/test/java/com/amazon/dataprepper/plugins/processor/drop/DropProcessorTests.java

```diff
@@ -0,0 +1,49 @@
+package com.amazon.dataprepper.plugins.processor.drop;
+
+import com.amazon.dataprepper.model.configuration.PluginSetting;
+import com.amazon.dataprepper.model.event.Event;
+import com.amazon.dataprepper.model.event.JacksonEvent;
+import com.amazon.dataprepper.model.record.Record;
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.Collections;
+
+public class DropProcessorTests {
+    private String messageInput;
+    private DropProcessor dropProcessor;
+    private PluginSetting pluginSetting;
+    private final String PLUGIN_NAME = "drop_events";
+
+    @Test
+    void doExecute() {
```

Comment:
It would be good to have a test which also has more than one item in the input list.

Reply:
Added the second test